std/string.d

```diff
@@ -5549,6 +5554,7 @@ S1 munch(S1, S2)(ref S1 s, S2 pattern) @safe pure @nogc
 }
 
 ///
+deprecated
 @safe pure @nogc unittest
```

Comment:
Yikes, why is this example publicly documented? We removed `munch`, so the example should go too